plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -392,13 +406,16 @@ def deploy(release_docs)
       Samson::Parallelizer.map(resources, db: true, &:deploy)
     end
 
-    def deploy_and_watch(release_docs, timeout)
+    def deploy_and_watch(release_docs, timeout, show_logs_on_success)
       deploy(release_docs)
       result = wait_for_resources_to_complete(release_docs, timeout)
       if result == true
         if blue_green = release_docs.select(&:blue_green_color).presence
           finish_blue_green_deployment(blue_green)
         end
+        if show_logs_on_success
+          show_pods_logs
```

Comment:
maybe even `show_pods_logs_if_requested`

Reply:
Yeah, makes sense! I was in doubt, just because of this (it maybe shows logs), but didn't come up with a better name. Thanks!

I changed the func to that name, and changed the variable to `show_logs_if_requested`.

Now the conditional on the var name and on the function name make me doubt, but well... naming is hard. Let me know what you think :)